app/vcap_services.py

```diff
@@ -40,6 +40,12 @@ def get_database_uri():
         hostname = creds["hostname"]
         port = creds["port"]
         name = creds["name"]
+    elif 'TRAVIS' in os.environ:
+        username = 'root'
+        password = ''
```

Comment:
SInce all the variables are the same, can we just do another `if 'Travis'` after the initial `if/else` that just changes the password?